src/components/style/EventPlotItem.scss

```diff
@@ -30,10 +30,18 @@
 
         &.expanded > .children {
             display: block;
+            -webkit-box-shadow: inset -8px -8px 6px -6px  rgba(0,0,0,0.33), inset -8px 8px 6px -6px  rgba(0,0,0,0.26);
+            -moz-box-shadow: inset -8px -8px 6px -6px  rgba(0,0,0,0.33), inset -8px 8px 6px -6px  rgba(0,0,0,0.26);
```

Comment:
Are those vendor prefixes still needed? I suspect browsers that old will not be able to render this project for many other reasons.